app/views/projects/samples/attachments/_new_attachment_modal.html.erb

```diff
@@ -1,21 +1,9 @@
-<%= turbo_frame_tag "new_attachment_modal" do %>
-  <%= viral_dialog(open: open) do |dialog| %>
-    <% dialog.with_trigger do %>
-      <%= viral_button(data: { action: "click->viral--dialog#open" }) do %>
-        <div class="flex">
-          <%= viral_icon(name: :plus, classes: 'h-5 w-5') %>
-          <span class="ml-2 grow">
-            <%= t("projects.samples.show.upload_files") %>
-          </span>
-        </div>
-      <% end %>
-    <% end %>
-    <% dialog.with_header(title: t("projects.samples.show.upload_files")) %>
-    <% dialog.with_section do %>
-      <%= render partial: "projects/samples/attachments/form",
-      locals: {
-        attachment: attachment
-      } %>
-    <% end %>
+<%= viral_dialog(open: open) do |dialog| %>
+  <% dialog.with_header(title: t("projects.samples.show.upload_files")) %>
+  <% dialog.with_section do %>
+    <%= render partial: "projects/samples/attachments/form",
```

Comment:
Before rendering the form, I would put a message that includes the names of the files that the user is about to concatenate.  Also maybe explain what the "Basename" is.

Reply:
The files display is being added in the next pr after this one

Reply:
Ok, as long as it gets updated in the next one.